data/presets/man_made/pier.json

```diff
@@ -25,7 +25,6 @@
         "jetty",
         "landing",
         "promenade",
-        "quay",
```

Comment:
'Quay' is not a synonym for `pier'. ('dock' however is US English for both ''pier' and 'quay', so both have it.)

Reply:
Strictly speaking, the `terms` are not only for "synonyms", but for all kinds of search terms a user might type when searching for a particular thing. As piers are often found near quays, it might be acceptable to continue to include it as a search term here…

But I'm also fine with dropping it.

Reply:
My guess is that people might search for 'dock' or even 'pier' when they want a quay, but not the other way around ('quay' being a somewhat more difficult word in terms of vocabulary).

'dock' should definitely be in both.